modules/dnn/src/layers/batch_norm_layer.cpp

```diff
@@ -432,6 +432,18 @@ class BatchNormLayerImpl CV_FINAL : public BatchNormLayer
     }
 #endif
 
+    virtual bool tryQuantize(const std::vector<std::vector<float> > &scales,
```

Comment:
What's the reason to add this `tryQuantize` for batch norm layer and others?

Reply:
Hi Ningxin, this `tryQuantize` function seems to be a newly added function in the main branch.

Reply:
When I upload my new codes, github reminds me that there are some conflicts since the main branch added this `tryQuantize` function but my PR didn't. Thus, in order to resolve the possible conflicts, I added this function to my PR.

Reply:
this comes from upstream merged PR #20228 (some experimental whole network int8 quantization...)